src/controllers/beneficial.owner.other.controller.ts

```diff
@@ -12,7 +12,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session);
 
     return res.render(config.BENEFICIAL_OWNER_OTHER_PAGE, {
-      backLinkUrl: config.BENEFICIAL_OWNER_TYPE_PAGE,
+      backLinkUrl: config.BENEFICIAL_OWNER_TYPE_URL,
```

Comment:
Unrelated, but noticed that this should really be a URL not a 'page' so changed it. Tested and working fine.